app/models/user.rb

```diff
@@ -41,7 +56,20 @@ def create_reset_token
     
     self.reset_token
   end
-  
+   
+  def activate 
+    self.update_attribute(:activated, true)
+    self.update_attribute(:activated_at, Time.zone.now)
```

Comment:
Replace this with:
```ruby
update_attributes(
  activated: true,
  activated_at: Time.now
)
```

You don't have to use `self.` unless you have conflicting variable names (a `activated` variable and the `self.activated` attribute) or if you're setting a specific attribute (`self.activated = true`).